bot/bot.py

```diff
@@ -45,6 +47,17 @@ def member(self) -> Optional[discord.Member]:
             return None
         return guild.me
 
+    @Cog.listener()
+    async def on_thread_join(self, thread: Thread) -> None:
+        """
+        Try to join newly created threads.
+
+        Despite the event name being misleading, this is dispatched when new threads are created.
+        We want our bots to automatically join threads in order to answer commands using their prefixes.
+        """
+        with suppress(Forbidden):
+            await thread.join()
```

Comment:
```suggestion
        if thread.me:
           # Already in this thread, return early
           return

        with suppress(Forbidden):
            await thread.join()
```
on_thread_join is triggered on creation of a thread, and when ever someone joins a thread.

Since `thread.join()` makes an API request, I think we should check if the bot is already in the thread before trying to join it again.